tuna/worker_interface.py

```diff
@@ -91,9 +93,12 @@ def __init__(self, **kwargs):
     self.end_jobs = None
     #job detail vars
     self.envmt: List = []
-    self.fetch_state: List = ['new']
+    self.fetch_state = set()
```

Comment:
Set is a fair idea for this var. The type hint for get_job should be updated to reflect this.